components/buy/Buy.vue

```diff
@@ -76,7 +80,7 @@ const handleBuy = async (nfts: TokenToBuy | TokenToBuy[]) => {
       interaction: ShoppingActions.BUY,
       nfts,
       urlPrefix: urlPrefix.value,
-      successMessage: $i18n.t('mint.successNewNfts'),
+      successMessage: null,
```

Comment:
What if we pass another parameter to trigger a bigger notification component instead of `null`?

Reply:
sure